### PR TITLE
[Bugfix] ensure global does not leak

### PR DESCRIPTION
**Pull request by stefanpenner**
Now the application will remove its global on destruction.

### CODE REVIEW

File: tests/unit/initializers/export-application-global-test.js [hunk 1]
```diff
@@ -27,17 +27,19 @@ module('ExportApplicationGlobalInitializer', {
...
+  var app = { reopen: function(){} };
+  initialize(null, app);
 
-  assert.equal(window.Foo, 'blazorz');
+  assert.equal(window.Foo, app);
 });
 
 test('it does not set the global unless exportApplicationGlobal is true', function(assert) {
   config.modulePrefix = 'foo';
   config.exportApplicationGlobal = false;
```
Comment (pixelhandler) [inline tests/unit/initializers/export-application-global-test.js]: is there already a test that does the opposite `exportApplicationGlobal = true;` ?

Comment (stefanpenner) [inline tests/unit/initializers/export-application-global-test.js]: now we are: https://github.com/ember-cli/ember-export-application-global/commit/9c12bfe21cc79f164b32b96872794d13bde40e01

Comment (pixelhandler) [inline tests/unit/initializers/export-application-global-test.js]: yeah that was my curiosity, thanks :)

Comment (stefanpenner) [inline tests/unit/initializers/export-application-global-test.js]: :)

**Pull request merged**